tests/test-is-cookie-domain-acceptable.c

```diff
@@ -131,7 +131,11 @@ int main(int argc, const char * const *argv)
 
 		if (valgrind && *valgrind) {
 			size_t cmdsize = strlen(valgrind) + strlen(argv[0]) + 32;
+#ifndef HAVE_ALLOCA_H
```

Comment:
I think we can just drop using alloca() and always use malloc.
If you don't mind, I'll prepare a patch for it and put you in as `Reported-by:`.

Reply:
Works for me.